controllers/goflowkube/goflowkube_objects.go

```diff
@@ -93,22 +111,37 @@ func buildMainCommand(desired *flowsv1alpha1.FlowCollectorGoflowKube) string {
 
 func buildConfigMap(desiredGoflowKube *flowsv1alpha1.FlowCollectorGoflowKube,
 	desiredLoki *flowsv1alpha1.FlowCollectorLoki, ns string) *corev1.ConfigMap {
-	data := fmt.Sprintf(`
-{
-listen: "netflow://:%d",
-loki: {
-	url: %s,
-	labels: ["SrcNamespace","SrcWorkload","DstNamespace","DstWorkload"],
-}}
-`, desiredGoflowKube.Port, desiredLoki.URL)
+
+	configStr := `{}`
+	config := &ConfigMap{
```

Comment:
maybe we could use the model from goflow-kube (importing `"github.com/netobserv/goflow2-kube-enricher/pkg/config"`) (if that doesn't pull too many dependencies) ?

Reply:
That's fair but it will need conversions or at least change for `FlowCollectorLoki` types :
- time.Duration / metav1.Duration
- model.LabelSet / map[string]string)
- int64 & int32 / int

Reply:
(never mind, let's keep duplicating the model to avoid any dependency hell!)